pages/linux/cockpit-ws.md

```diff
@@ -0,0 +1,28 @@
+# cockpit-ws
+
+> Used for communication between the browser application and various configuration tools and services like cockpit-bridge.
+> More information: <https://cockpit-project.org/guide/latest/cockpit-ws.8.html>.
+
+- To check which certificate cockpit-ws will use:
```

Comment:
```suggestion
- Check which certificate `cockpit-ws` will use:
```
I am not sure if this command is fine as it isn't a part of `cockpit-ws` we don't usually add external commands to these pages. @navarroaxel What do you think?

Reply:
Exactly, this correspond to new page called `remotectl`

Reply:
Should I remove it and push again?

Reply:
I am new to Open Sourcing and don't know how these "request reviews" work. Pardon me if Ive done anything wrong.
Like mentioned I have removed the remotectl command and pushed again.